app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -37,6 +46,9 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+
+
+
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/8d3333d0ff432d0557742aed5ff0615effcedf8c/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L53"></span>Parameter item should be final.</td>
    </tr>
  </tbody>
</table>